### PR TITLE
Fix the Foresee 404 error on vagovdev or development builds 

### PR DESCRIPTION
**Pull request by accbjt**
## Description
Fix the Foresee 404 error on vagovdev or development builds 

## Testing done
Local Test

## Acceptance criteria
- [ ] The Foresee script should only render on the 'preview' or 'vagovstaging' environments.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: content/includes/survey-tools.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% assign auth_required_entrynames = 'claims-status, letters, health-records, messaging, rx, profile-360' | split: ', ' %}
 {% if auth_required_entrynames contains entryname  %}
-  <script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+  {% if buildtype == 'vagovstaging' or buildtype == 'preview' %}
+    <script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+  {% endif %}
```
Comment (ncksllvn) [inline content/includes/survey-tools.html]: The `content` directory is only for Vets.gov, so I don't think we have to touch this file.

File: va-gov/includes/survey-tools.html [hunk 2]
```diff
@@ -1 +1,3 @@
-<script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+{% if buildtype == 'vagovstaging' or buildtype == 'preview' %}
+  <script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+{% endif %}
```
Comment (ncksllvn) [inline va-gov/includes/survey-tools.html]: 👍This also reminds me that I think we need Foresee script, `vagovprod.js`, to configure Foresee on `www.va.gov`.

CC: @BBarone1

Comment (BBarone1) [reply va-gov/includes/survey-tools.html]: thanks @ncksllvn is that something that we can add here or do we need a separate implementation ticket for a new PR?

Comment (accbjt) [reply va-gov/includes/survey-tools.html]: @ncksllvn Do we want to do this with this ticket or create a new ticket. If we are doing this in this ticket who is the one that is going to implement it. I am not familiar with the Foresee service so I would need some documentation if I were to implement it.

Comment (jbalboni) [reply va-gov/includes/survey-tools.html]: I think we just need to make a copy of the preview one named vagovprod.js. The current preview one has the production, va.gov changes in it.

File: va-gov/includes/survey-tools.html [hunk 3]
```diff
@@ -1 +1,3 @@
-<script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+{% if buildtype == 'vagovstaging' or buildtype == 'preview' %}
```
Comment (jbalboni) [inline va-gov/includes/survey-tools.html]: I think this also needs to include `vagovprod`

**Review by jbalboni — APPROVED** [review]

**Pull request merged**